dist/common/nodetool-completion

```diff
@@ -0,0 +1,269 @@
+# Apache Cassandra
+# Copyright 2009- The Apache Software Foundation
+#
+# This product includes software developed by The Apache Software
+# Foundation (http://www.apache.org/).
+#
+# Modified by ScyllaDB
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+have nodetool && have cqlsh &&
```

Comment:
Curious, what is `have` here?

Reply:
I had to look up because I also didn't know. According to https://stackoverflow.com/questions/12874920/have-keyword-for-bash-completion, it is just a convenience wrapper to check for existence of certain executables.

Reply:
`have` is a bash function, which checks for the existence of a certain command. so that the code below is evaluated only if `nodetool` and `cqlsh` are in `$PATH`. see `/usr/share/bash-completion/bash_completion` line 107 .